.secrets.baseline

```diff
@@ -87,24 +87,6 @@
         "verified_result": null
       }
     ],
-    "oem/ibm/pfw-sms-utils/pfw_sms_menu.cpp": [
```

Comment:
why was this removed?

Reply:
+1

Reply:
Rather .secrets.baseline is a generated file ? For this commit to I think that file is not needed.

Reply:
okk

Reply:
But this commit has no change other than what running the local CI caused, so I added it. Please let me know how to handle this for CI to pass?

Reply:
Riya, you can ignore it as that deleted pfw file is not yet present in 1110-ghe and it will only be checked in with my SMS changes. Secrets file was just added for CI to pass automatically. So I am good here.

Reply:
cool, sounds good then.